agent/api/task_test.go

```diff
@@ -174,99 +174,6 @@ func TestDockerHostConfigVolumesFrom(t *testing.T) {
 	}
 }
 
-func TestDockerHostConfigRawConfig(t *testing.T) {
```

Comment:
Instead of moving the actual tests, can you just define constants that are different for Windows and Linux and keep the tests here?

Reply:
Done